### PR TITLE
Discard IMU/Img msgs if not strictly increasing timestamps

### PR DESCRIPTION
**Pull request by mfehr**
Changes:
 - Add logic to all datasources in ROVIOLI and maplab_node to discard and issue a warning if IMU messages or image message are not strictly increasing. This does not happen with a decent sensor setup, but if it does happen for some reason it currently occasionally crashes the nodes.
- Two tiny unrelated fixes are also part of this PR
   - fix RGBDI sensor launch file, rm duplicate calibration file
   - Add forgotten console plugin to the maplab console meta package.

@b-hahn @smauq let me know if the way it is solved here also works for you. If I remember correctly in the wheel odom PR there was some similar stuff.

### CODE REVIEW

File: applications/rovioli/src/datasource-rosbag.cc [hunk 1]
```diff
@@ -45,7 +45,12 @@ DataSourceRosbag::DataSourceRosbag(
     : shutdown_requested_(false),
       all_data_streamed_(false),
       rosbag_path_filename_(rosbag_path_filename),
-      ros_topics_(ros_topics) {
+      ros_topics_(ros_topics),
+      last_imu_timestamp_ns_(aslam::time::getInvalidTime()) {
+  const uint32_t num_cameras = ros_topics_.camera_topic_cam_index_map.size();
```
Comment (LBern) [inline applications/rovioli/src/datasource-rosbag.cc]: 32 bits for `num_cameras` is quite optimistic. i don't think we will ever have 4 billion cameras ;)  i believe 8 bits are enough here

File: applications/maplab_node/src/datasource-rostopic.cc [hunk 2]
```diff
@@ -23,7 +23,13 @@ DataSourceRostopic::DataSourceRostopic(
     : DataSource(sensor_manager),
       shutdown_requested_(false),
       ros_topics_(ros_topics),
-      image_transport_(node_handle_) {} 
+      image_transport_(node_handle_),
+      last_imu_timestamp_ns_(aslam::time::getInvalidTime()) {
+  const uint32_t num_cameras = ros_topics_.camera_topic_cam_index_map.size();
```
Comment (LBern) [inline applications/maplab_node/src/datasource-rostopic.cc]: see other comment

File: applications/rovioli/src/datasource-rostopic.cc [hunk 3]
```diff
@@ -17,7 +17,12 @@ DataSourceRostopic::DataSourceRostopic(
     const vio_common::RosTopicSettings& ros_topics)
     : shutdown_requested_(false),
       ros_topics_(ros_topics),
-      image_transport_(node_handle_) {
+      image_transport_(node_handle_),
+      last_imu_timestamp_ns_(aslam::time::getInvalidTime()) {
+  const uint32_t num_cameras = ros_topics_.camera_topic_cam_index_map.size();
```
Comment (LBern) [inline applications/rovioli/src/datasource-rostopic.cc]: `uint8_t`

File: applications/maplab_node/src/datasource-rosbag.cc [hunk 4]
```diff
@@ -50,7 +50,13 @@ DataSourceRosbag::DataSourceRosbag(
       shutdown_requested_(false),
       all_data_streamed_(false),
       rosbag_path_filename_(rosbag_path_filename),
-      ros_topics_(ros_topics) {
+      ros_topics_(ros_topics),
+      last_imu_timestamp_ns_(aslam::time::getInvalidTime()) {
+  const uint32_t num_cameras = ros_topics_.camera_topic_cam_index_map.size();
```
Comment (LBern) [inline applications/maplab_node/src/datasource-rosbag.cc]: one more time

**Comment by mfehr** [comment]
@smauq or @b-hahn feel free to add it to this PR, Alice doesn't need this PR urgently, but it would add some more robustness. I was just processing some RGBDI datasets and due to some weird rosbag splitting issue this stuff was needed.

**Comment by smauq** [comment]
Ok, I'll do it in a bit. It's a good fix to have

**Comment by smauq** [comment]
Should be done now, and I tested on a mobmi bag that everything still seems to be running with the changes I added on top

**Pull request merged**